integreat_cms/static/src/js/pages/page-clickable-sidebar.ts

```diff
@@ -0,0 +1,23 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const boxes = document.querySelectorAll(".sidebar-box");
+    const cookies = document.cookie.split("; ").map((cookie) => cookie.split("="));
+    for (let i = 0; i < boxes.length; i++) {
+        boxes[i].firstElementChild.addEventListener("click", () => {
+            boxes[i].lastElementChild.classList.toggle("hidden");
+            let cookieValue = "hidden";
+            if (boxes[i].lastElementChild.classList.contains("hidden")) {
+                cookieValue = "hidden";
+            } else {
+                cookieValue = "show";
+            }
```

Comment:
```suggestion
            const cookieValue = boxes[i].lastElementChild.classList.contains("hidden") ? "hidden" : "show";
```
is a bit cleaner/more readable, I think